src/Confluent.Kafka/include_docs.xml

```diff
@@ -0,0 +1,35 @@
+<API>  
+
+<Member name="ISerializer_Configure">  
+    <summary>
+        Configure the serializer using relevant configuration parameter(s) in <paramref name="config" /> (if present)
```

Comment:
the serializer can use whatever it wants - generally not librdkafka config properties, but the possibility is there... i think this is ok.